example/storybook/src/components/Forms/FormControl/index.stories.mdx

```diff
@@ -275,6 +301,607 @@ The following are the lists of all the libraries and packages the component reli
+          </Button>
+        </HStack>
+        <FormControl.Error mt='$2'>
+          <FormControl.Error.Text fontSize='$sm' lineHeight='$md'>
+            Choose only one time slot for the meeting
+          </FormControl.Error.Text>
+        </FormControl.Error>
+      </FormControl>
+      `,
+      transformCode: (code) => {
+        return transformedCode(code);
+      },
+      scope: {
+        Wrapper,
+        FormControl,
+        Icon,
+        HStack,
+        ChevronDownIcon,
+        Input,
+        Select,
+        Button,
+      },
+      argsType: {},
+    }}
+  />
+</AppProvider>
+
+#### Form Control with Form in Modal
+
+<AppProvider>
+  <CodePreview
+    showComponentRenderer={true}
+    showArgsController={false}
+    metaData={{
+      code: `
+      function App() {
+        const [showModal, setShowModal] = useState(false);
+        return(
+          <Wrapper>
+            <Center h={500}>
+              <Button
+                py='$2.5'
+                px='$3'
+                bg='$darkBlue600'
+                onPress={()=>{
+                  setShowModal(true);
+                }}
+              >
+                <Button.Text color='$text50' fontSize='$sm' fontWeight='$medium'>
+                  Change Settings
+                </Button.Text>
+              </Button>
+              <Modal
+                isOpen={showModal}
+                onClose={() => {
+                  setShowModal(false)
+                }}
+              >
+                <Modal.Content maxWidth='$96' bg='$white'>
+                  <Modal.Body p='$5'>
+                    <VStack space='xs' mb='$4'>
+                      <Heading lineHeight='$' color='$text900'>
+                        Change Settings
+                      </Heading>
+                      <Text size='sm' color='$text900'>
+                        Make modifications to the settings with ease.
+                      </Text>
+                    </VStack>
+                    <FormControl>
+                      <VStack py='$2' space='xl'>
+                        <VStack space='xs'>
+                          <FormControl.Label.Text color='$text500'>
+                            Name
+                          </FormControl.Label.Text>
+                          <Input>
+                            <Input.Input
+                              type="text"
+                              py='$2'
+                              px='$3'
+                              value='Wade Warren'
+                            />
+                          </Input>
+                        </VStack>
+                        <VStack space='xs'>
+                          <FormControl.Label>
+                            <FormControl.Label.Text color='$text500'>
+                              Email
+                            </FormControl.Label.Text>
+                          </FormControl.Label>
+                          <Input>
+                            <Input.Input
```

Comment:
we shouldn't explicitly provided the styled that are already present in the theme.